openlibrary/plugins/upstream/models.py

```diff
@@ -413,14 +413,19 @@ def wp_citation_fields(self):
         result = {
             "title": self.works[0].title.replace("[", "&#91").replace("]", "&#93"),
             "publication-date": self.get('publish_date'),
-            "url": "http://openlibrary.org%s" % self.url()
+            "ol": str(self.get_olid()),
```

Comment:
The instructions on Wikipedia say to drop the OL from the value here. I tested the citation format and both with and without the OL link to the correct place. I'd prefer the full id as the value, but it is Wikipedia's citation format :)